cmd/gb-vendor/vendor/_testdata/src/github.com/foo/bar/main.go

```diff
@@ -0,0 +1,12 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/quux/flobble"
+	// "bitbucket.org/fwoop/ftang" // commented out, this is deliberate
```

Comment:
We could also add something like `import flobble "github.com/quux/flobble"` or `import _ "github.com/quux/flobble"`.